app-text/pandoc/pandoc-2.14.1.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.7.9999
+#hackport: flags: -static
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Conversion between markup formats"
+HOMEPAGE="https://pandoc.org"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="embed-data-files trypandoc"
+
```

Comment:
Do the tests now pass? There is a test restriction from pandoc-2.13 that may need to continue to apply.

Reply:
2/2821 tests fail, and I've opened up a ticket about this:
jgm/pandoc/issues/7449

I'm hoping there will be a resolution so that we can leave the tests enabled.